gigareview/common/tests/unit/ManuscriptTest.php

```diff
@@ -32,4 +32,66 @@ public function testCanCreateTimestamp()
+            $this->assertEquals($sampleCsvReportData[0]['article_title'], $instance->article_title, "Value is not matched in Manuscript instance!");
+            $this->assertEquals($sampleCsvReportData[0]['editorial_status_date'], $instance->editorial_status_date, "Value is not matched in Manuscript instance!");
+            $this->assertEquals($sampleCsvReportData[0]['editorial_status'], $instance->editorial_status, "Value is not matched in Manuscript instance!");
+        }
+    }
+
+    public function testCanCreateInstancesFromReportWithTwoEntries()
+    {
+        $sampleCsvReportData = [
+            [
+                'manuscript_number' => 'GIGA-D-22-00054',
+                'article_title' => 'A machine learning framework for discovery and enrichment of metagenomics metadata from open access publications',
+                'editorial_status_date' => '6/7/2022',
+                'editorial_status' => 'Final Decision Accept'
+            ],
+            [
+                'manuscript_number' => 'GIGA-D-22-00060',
+                'article_title' => 'A chromosome-level genome of the booklouse, Liposcelis brunnea provides insight into louse evolution and environmental stress adaptation',
+                'editorial_status_date' => '6/7/2022',
+                'editorial_status' => 'Final Decision Reject'
+            ],
+        ];
+
+        $manuscriptInstances = Manuscript::createInstancesFromEmReport($sampleCsvReportData);
+        $this->assertNotNull($manuscriptInstances);
+        $this->assertIsArray($manuscriptInstances);
+
+        for ($i=0; $i <= count($sampleCsvReportData) - 1; $i++) {
+            $this->assertInstanceOf('common\models\Manuscript', $manuscriptInstances[$i], "Manuscript Instance is not instantiated!");
+            $this->assertArrayHasKey('manuscript_number', $manuscriptInstances[$i], "Key is not found in Manuscript instance!");
+            $this->assertArrayHasKey('article_title', $manuscriptInstances[$i], "Key is not found in Manuscript instance!");
+            $this->assertArrayHasKey('editorial_status_date', $manuscriptInstances[$i], "Key is not found in Manuscript instance!");
+            $this->assertArrayHasKey('editorial_status', $manuscriptInstances[$i], "Key is not found in Manuscript instance!");
+            $this->assertEquals($sampleCsvReportData[$i]['manuscript_number'], $manuscriptInstances[$i]->manuscript_number, "Value is not matched in Manuscript instance!");
+            $this->assertEquals($sampleCsvReportData[$i]['article_title'], $manuscriptInstances[$i]->article_title, "Value is not matched in Manuscript instance!");
+            $this->assertEquals($sampleCsvReportData[$i]['editorial_status_date'], $manuscriptInstances[$i]->editorial_status_date, "Value is not matched in Manuscript instance!");
+            $this->assertEquals($sampleCsvReportData[$i]['editorial_status'], $manuscriptInstances[$i]->editorial_status, "Value is not matched in Manuscript instance!");
```

Comment:
All the messages are the same. The messages could contain the key and/or value from the associate array for a more informative message.